contracts/pool/concentrated/ConcentratedLiquidityPool.sol

```diff
@@ -123,10 +124,11 @@ contract ConcentratedLiquidityPool is IPool {
             _deployData,
             (address, address, uint24, uint160, uint24)
         );
-
+        if (TickMath.MIN_SQRT_RATIO >= price && price >= TickMath.MAX_SQRT_RATIO) revert BadPrice();
```

Comment:
It is impossible for price to be greater than the max and less than the min.

Reply:
The price is set by the user, if this check doesn't exists they can set whatever price they want.

Reply:
You don't understand. It is impossible because TickMath.MIN_SQRT_RATIO < TickMath.MAX_SQRT_RATIO.

Reply:
Should be:
```
if (TickMath.MIN_SQRT_RATIO >= price || price >= TickMath.MAX_SQRT_RATIO) revert BadPrice();
```

Reply:
Ah, yes, you're right, I'll properly look at the issue, and send a PR with proper test. Thank you!

Reply:
I've updated it with actual _price variable now, it should work as expected.